versioned_docs/version-1.2-rc1(WIP)/layer2Account.md

```diff
@@ -0,0 +1,39 @@
+---
+id: layer2Account
+title: Layer 2 Account in Godwoken
+---
+
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+Godwoken v1 introduced the registry and registry address, both of which can be seen as alias mechanisms for layer 2 accounts. 
+
+A layer 2 account can be referenced by ID or by script (a hash of a script usually) itself. However, in some environments, it is also required to access an account by alias. For example, there is no such thing as a Godwoken `ID` or `script` in the EVM environment, and so access to an account can only be made through an ETH address.
+
+In this case, a registry is an account. The alias mapping relation is stored in the key-value storage of the registry account. The registry address is encoded as follows:
```

Comment:
In this case, a registry is an account. Alias mappings are stored in the key-value storage of the registry account. The registry address is encoded as follows: